src/main/java/uk/gov/justice/probation/courtcaseservice/jpa/entity/OffenceEntity.java

```diff
@@ -40,13 +40,6 @@
     @JsonIgnore
     private final Long id;
```

Comment:
I think we ought to be able to delete this entity as well though there is [another ticket ](https://dsdmoj.atlassian.net/browse/PIC-1795)to pick that up... not sure why we've split it into multiple tickets! Up to you how you want to do it.